pypeit/reduce.py

```diff
@@ -1029,24 +1076,20 @@ def find_objects_pypeline(self, image, std_trace=None,
 
             sobjs.add_sobj(sobjs_slit)
 
-        # If this is a second pass for objfind or this is the first one but skip_second_find is True,
-        # assign slitmask design information to detected objects
+        # If this is a second pass for objfind or this is the first one but skip_second_find is True, extract some
+        # useful info from slitmask design. Matching to the detected objects is done in local_skysub_extract
```

Comment:
Matching to the detected objects need to be done here, not in local_skysub_extract. I'm not sure why you are doing that. Object finding and extraction should be maintained as separate operations.